repose-aggregator/commons/utilities/src/test/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapperTest.scala

```diff
@@ -521,6 +523,7 @@ class HttpServletRequestWrapperTest extends FunSpec with BeforeAndAfter with Mat
       wrappedRequest.getPreferredHeader("ornament") shouldBe "santa"
     }
 
+    //todo: fix this test -- shouldn't the higher quality value be returned?
     it("should return the appropriate value when a higher quality is appended onto a line") {
```

Comment:
Yeah, that's kind of what I meant by that last block of text in my last comment. Still, going that route means the function breaks if we're given a supposedly non-splittable header that is actually multiple headers the earlier of which has parameters. I guess it doesn't matter because that use case kind of breaks the contract, but parsing from the right doesn't hurt as (as far as I know) so I figured we may as well do that.

Alas, do no care. Please don't break the contract when using the wrapper.